### ISSUE TITLE
Artifacts with pre-trained and cryoTIGER models

### ISSUE THREAD

**Issue by 3P1L** [turonova/cryoTIGER#1]
Dear cryoTIGER Team,

thanks for providing this great program.

I just came across an issue with the models for the demo data.

In my hands, they create significant artifacts (see attached screenshots).

Did you ever experience this before or do you have other models than the ones provided on the GH landing page?

Thanks!

![Image](https://github.com/user-attachments/assets/33ce78f9-fdd8-4dcc-ac50-a39d0116b0fd)
![Image](https://github.com/user-attachments/assets/3974ff2d-9209-414c-a244-bfa9bf048e48)

**Comment by majtner** [comment]
Hi there,

thank you for raising this issue.

We're aware of the 2D tilt visualization artifacts you're encountering—our processing sometimes produces similar results as well. These artifacts are primarily caused by two factors:

1. **Data size**: For this demonstration, we only uploaded 8x binned tilt series. When using a lower binning, these artifacts tend to disappear.

2. **Data conversions**: We apply a conversion to RGB (and back to grayscale) with normalizations, which can sometimes lead to artifacts, especially when displaying the tilt series.

That being said, the good news is that when you proceed with the reconstruction, including these tilts, you'll still see improved results, as outlined in the paper.

For context, here's a quick example of a reconstruction using **weighted back projection** as implemented in **Imod** (with an FSirt filter for better visualization):

```bash
tilt -THICKNESS 400 -FULLIMAGE 512,512 -MODE 1 -COSINTER 0 -FakeSIRTiteration 5 \
     -InputProjections tutorial_data/interpolated_TS/demo_tilt_series_bin8_no-int.mrc \
     -OutputFile tutorial_data/interpolated_TS/demo_tomo_bin8_no-int_tmp.mrc \
     -TILTFILE tutorial_data/rawtlt/0248_dose-filt.rawtlt \
     -SHIFT 0,0
```

This is also applied similarly for `demo_tilt_series_bin8_interpolated_cryoTIGER_model.mrc` and `demo_tilt_series_bin8_interpolated_pre-trained_model.mrc`. Below, you can see an example of the 2D slice from the 3D reconstructed volume, both with and without interpolation:

![Image](https://github.com/user-attachments/assets/7fd7efa7-8c4e-4e12-9e95-236696b15bb6)

We've also uploaded the `.rawtlt` files in case you'd like to try reconstructing this particular tomogram.

For practical applications, we recommend using **novaCTF** for tomogram reconstruction, as it takes into account 3D CTF information and is better suited for workflows such as subtomogram averaging.

**Weighted Back Projection (WBP)** is used here only as a quick example for demonstration purposes.

Let us know if you have any questions or need further clarification!
